models/POP/shell_scripts/cesm2_1/CESM2_1_setup_hybrid

```diff
@@ -0,0 +1,770 @@
+# just T,S. A hybrid start is also more desirable because initial values can be
+# specified for ROF - as opposed to just zeros.
+# ==============================================================================
+
+./xmlchange RUN_TYPE=hybrid
+./xmlchange RUN_STARTDATE=${start_year}-${start_month}-${start_day}
+./xmlchange START_TOD=$start_tod
+./xmlchange RUN_REFCASE=$refcase
+./xmlchange RUN_REFDATE=$refdate
+./xmlchange RUN_REFTOD=$reftod
+./xmlchange GET_REFCASE=FALSE
+./xmlchange EXEROOT=${EXEROOT}
+./xmlchange RUNDIR=${RUNDIR}
+
+./xmlchange DATM_MODE=CPLHIST
+./xmlchange DATM_CPLHIST_CASE=${CASE}
+./xmlchange DATM_CPLHIST_YR_ALIGN=$stream_year_align
+./xmlchange DATM_CPLHIST_YR_START=$stream_year_first
+./xmlchange DATM_CPLHIST_YR_END=$stream_year_last
+
+# The streams files were generated with a NO_LEAP calendar in mind.
+# We need to test these with a GREGORIAN calendar.
+./xmlchange CALENDAR=GREGORIAN
+
+./xmlchange STOP_OPTION=$stop_option
+./xmlchange STOP_N=$first_STOP_N
+./xmlchange CONTINUE_RUN=FALSE
+./xmlchange RESUBMIT=$resubmit
+
+./xmlchange PIO_TYPENAME=pnetcdf
+
+# These are archiving options that may be used.
+# You can turn the short/long term archivers on or off,
+# but these settings should be made in either event.
+
+./xmlchange DOUT_S_ROOT=${ARCHDIR}
+./xmlchange DOUT_S_SAVE_INTERIM_RESTART_FILES=TRUE
+
+# level of debug output, 0=minimum, 1=normal, 2=more, 3=too much, valid values: 0,1,2,3 (integer)
+
+./xmlchange DEBUG=FALSE
+./xmlchange INFO_DBUG=0
+
+# ==============================================================================
+# Updating source files.
+#
+# Ideally, using DART would not require any modifications to the model source.
+# Until then, this script accesses sourcemods from a hardwired location.
+# If you have additional sourcemods, they will need to be merged into any DART
+# mods and put in the SourceMods subdirectory found in the 'CASEROOT' directory.
+# ==============================================================================
+
+echo ""
+echo "Updating source files."
+echo ""
+
+if (    -d     ~/${cesmtag}/SourceMods ) then
+   ${COPY} -r  ~/${cesmtag}/SourceMods/* ${CASEROOT}/SourceMods/
+else
+   echo "ERROR - No SourceMods for this case."
+   echo "DART requires modifications to several src files."
+   echo "Download the appropriate files for CESM2_1 from:"
+   echo "http://www.image.ucar.edu/pub/DART/CESM"
+   echo "untar these into your HOME directory - they will create a"
+   echo "~/${cesmtag}  directory with the appropriate SourceMods structure."
+   exit -4
+endif
+
+# ==============================================================================
+# Updating the run script to reflect queue and wallclock
+# ==============================================================================
+
+echo ""
+echo "Updating the run script to set wallclock and queue."
+echo ""
+
+./xmlchange --subgroup case.st_archive --id JOB_WALLCLOCK_TIME --val $timewall
```

Comment:
I am putting in a comment here ... the short term archiver may take a LOT longer than the assimilation.
I am going to commit this as is, but we need to revisit it before creating a release.